src/components/maps/MapLayersSelector.tsx

```diff
@@ -0,0 +1,63 @@
+import * as Popover from '@radix-ui/react-popover'
+import { MAP_STYLES, type MapStyles } from './MapSelector'
+import { useState } from 'react'
+
+interface Props {
+  emit: (key: keyof MapStyles) => void
+}
+
+const MapLayersSelector: React.FC<Props> = ({ emit }) => {
+  const [mapImgUrl, setMapImgUrl] = useState<string>(MAP_STYLES.standard.imgUrl)
+  const [mapName, setMapName] = useState<string>('standard')
+
+  const emitMap = (key: string): void => {
+    const styleKey = key as keyof MapStyles
+    const imgUrl = MAP_STYLES[styleKey].imgUrl
+
+    setMapName(key)
+    setMapImgUrl(imgUrl)
+    emit(styleKey)
+  }
+  return (
+    <>
+      <Popover.Root>
+        <Popover.Trigger asChild>
+          <button className='absolute bottom-20 left-3 bg-white rounded p-1.5 shadow'>
```

Comment:
Minor issues.  Instead of absolute colors, please use [DaisyUI color names](https://daisyui.com/docs/colors/) whenever you can (sorry this is not well-documented).  
- bg-white --> `bg-base-100`
- border-slate-300 --> `border-base-300` or `border-content/60` (play with the opacity to achieve the desired effect)